### PR TITLE
Update tours

### PR DESCRIPTION
**Pull request by mmiladi**

### CODE REVIEW

**Comment by mmiladi** [comment]
Please do not merge this PR now

File: tours/graphclust_wf.yaml [hunk 1]
```diff
@@ -138,6 +138,6 @@ steps:
 
 
 
-    - title: "<b>A tutorial on GrapClust workflow</b>"
+    - title: "<b>A tutorial on GarpHclust workflow</b>"
```
Comment (bgruening) [inline tours/graphclust_wf.yaml]: s/H/h/

File: tours/graphclust_wf.yaml [hunk 2]
```diff
@@ -75,7 +75,7 @@ steps:
 
     - title: "<b>Data Acquisition</b>"
       element: "#current-history-panel > div.controls"
-      intro: "You've acquired your data. Now let's call the <b>GrapClust Workflow</b>.<br><br>"
+      intro: "You've acquired your data. Now let's call the <b>GarpHclust Workflow</b>.<br><br>"
```
Comment (bgruening) [inline tours/graphclust_wf.yaml]: s/H/h/

File: tours/graphclust_wf.yaml [hunk 3]
```diff
@@ -29,7 +29,7 @@ steps:
 
     - title: "<b>Rename the history</b>"
       element: "#current-history-panel > div.controls"
-      intro: "Change the name of the new history to <b>'GrapClust'</b>."
+      intro: "Change the name of the new history to <b>'GarpHclust'</b>."
```
Comment (bgruening) [inline tours/graphclust_wf.yaml]: s/H/h/

**Pull request merged**